### PR TITLE
Generated more api types from swagger spec

### PR DESCRIPTION
**Pull request by dnephin**
Follow up to #27117 

Generate a bunch more types from the swagger spec, including some responses structs.

### CODE REVIEW

**Comment by dnephin** [comment]
rebased, and build is green

File: api/types/registry/authenticate.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package registry
+
+// ----------------------------------------------------------------------------
+// DO NOT EDIT THIS FILE
+// This file was generated by `swagger generate operation`
+//
+// See hack/swagger-gen.sh
+// ----------------------------------------------------------------------------
+
+// AuthenticateOKBody authenticate o k body
```
Comment (tiborvass) [inline api/types/registry/authenticate.go]: `o k` lol

**Review by vdemeester — APPROVED** [review]
LGTM 👼

File: api/swagger.yaml [hunk 2]
```diff
@@ -1063,17 +1063,28 @@ definitions:
...
     required: ["message"]
     properties:
       message:
         description: "The error message."
         type: "string"
+        x-nullable: false
     example:
       message: "Something went wrong."
 
+  IdResponse:
```
Comment (MHBauer) [inline api/swagger.yaml]: Nice idea. Reuse.

**Review by MHBauer — COMMENTED** [review]
Why suffix with Body vs Request? Seems inconsistent with others that suffix with Response. I think this is an artifact of the swagger generator.

**Comment by thaJeztah** [comment]
oh, needs a rebase @dnephin

**Comment by dnephin** [comment]
> Why suffix with Body vs Request?

Yes, the name convention from swagger is `<operation><response type><response field>`.

`Body` is arguably more correct because the response include headers and an error code, which are not part of our old `Response` structs. They will all be consistent once they are generated from the swagger spec.

**Comment by dnephin** [comment]
rebased and build is green

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**